browser/main/NoteList/index.js

```diff
@@ -349,6 +353,71 @@ class NoteList extends React.Component {
     })
   }
 
+  getSearchNotes () {
+    let { data } = this.props
+    let search = document.getElementsByClassName('TopBar__control-search-input___browser-main-TopBar-')[0].childNodes[0].value
```

Comment:
The class name depends on webpack, so you have to change it. If I were you, I name the element such as `searchInput`.